demo/api.html

```diff
@@ -24,26 +225,51 @@ <h1 id="basics" class="page-header">
         </p>
         <p>
           The full list of events in call priority:
-          <span class="label label-warning">any</span> [
+          <span class="label label-warning">update</span> [
           <span class="label label-danger">leave</span> >
           <span class="label label-success">match</span> (
           <span class="label label-primary">enter</span> >
           <span class="label label-info">change</span> ) ].
         </p>
-        <h2>enter</h2>
+        <h3 id="rule-event-enter">
+          enter
+          <a href="rule-event-enter" class="anchorjs-link"><span class="anchorjs-icon"></span></a>
+        </h3>
         <p>
           Called when you match a rule once it wasn't triggered before.
```

Comment:
I rephrased this one, hope it's clear now, let me know if you think I should work more on it. Thanks